GraphterEffectsTool/src/test/java/com/github/meteoorkip/util/XMLLoaderTest.java

```diff
@@ -0,0 +1,134 @@
+package com.github.meteoorkip.util;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+import testutils.TestUtils;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Tests functionality of XMLLoader
+ */
+public class XMLLoaderTest {
+
+
+    /**
+     * XML test file containing an xml root and nothing else.
+     */
+    private String only_root = null;
+    /**
+     * XML test file containing an xml root and nested children.
+     */
+    private String xml_with_children = null;
+    /**
+     * XML test file containing an xml root that has attributes and no children.
+     */
+    private String xml_with_attributes = null;
+
+    /**
+     * XMLLoader instance to act as Unit Under Test
+     */
+    private XMLLoader xmlLoader = null;
```

Comment:
Inconsistent, you probably want to use either underscores everywhere or camelCase everywher